### DISCUSSION TITLE
工作空间是什么东西，它是干什么的，没有他行不行，如果想自定义工作空间，怎么写？

### DISCUSSION THREAD

**Discussion by 136861535** [eclipse-emfcloud/emfcloud#137] [ModelServer]
请教一下，工作空间是什么东西，它是干什么的，没有他行不行，如果想自定义工作空间，怎么写？谢谢。

**Comment by eneufeld** [comment]
The workspace is the visible folder you are working in, the same as in vscode. The workspace thus is a subset of your file system.
Depending on your requirements you might not need a workspace.
The coffee-editor is relying on the workspace as it is used by the modelserver to track relevant files. So if you want to get rid of it you need to do quite some adaptations.

Please post your questions in english as you will help others to find your question and possible answers to it. Thank you.
